jsk_pcl_ros/include/jsk_pcl_ros/bilateral_filter.h

```diff
@@ -38,7 +38,7 @@
 #define JSK_PCL_ROS_BILATERAL_FILTER_H_
 
 #include <pcl_ros/publisher.h>
-#include <jsk_topic_tools/connection_based_nodelet.h>
+#include <jsk_topic_tools/diagnostic_nodelet.h>
```

Comment:
if this code depends on jsk_topic_tools >= x.x.x, I would recommend to write something like https://github.com/PR2/pr2_mechanism/commit/c2d2d7e0f18dad9f82f4f22fbc681d02b3aa6469#diff-b32b543817fc81911592c454ab29c1d29f6ce7cd2ab6dcd3a374014568a2ee55R81-R85

instead of using  `-DJSK_TOPIC_TOOLS_X_X_XAPI` , will we introduce jsk_topic_tools_version.h ???(https://github.com/jsk-ros-pkg/jsk_common/blob/714713378fb037c9670bf48118010f1899c10bd5/jsk_topic_tools/CMakeLists.txt#L30-L32)